locale/fa/site.json

```diff
@@ -1,9 +1,14 @@
 {
```

Comment:
If you remove the "blog" key here, it won't show up in the local navigation.
https://github.com/nodejs/nodejs.org/blob/4dff3118322ded0313efc5bc0db2460ef8c934c1/locale/fa/site.json#L138